src/components/organisms/AdminVenueView/components/Timing/Timing.scss

```diff
@@ -3,7 +3,7 @@
 $left-section-width: 360px;
 
 .Timing {
-  height: 100%;
+  height: calc(100% - #{$admin-navbar-height});
```

Comment:
NOTE: a variable of the kind of $admin-navbar-height should probably be used by all sibling tabs as to provide for uniformed UI across the admin.